internal/controllers/orano2ims_controller.go

```diff
@@ -224,6 +222,108 @@ func (r *ORANO2IMSReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	return
 }
 
+func (r *ORANO2IMSReconciler) createClientServiceAccount(ctx context.Context,
+	orano2ims *oranv1alpha1.ORANO2IMS) error {
+	_, err := r.createServiceAccount(ctx, orano2ims, utils.ORANO2IMSClientSAName)
+	return err
+}
+
+func (r *ORANO2IMSReconciler) createDeploymentManagerServiceAccount(ctx context.Context,
+	orano2ims *oranv1alpha1.ORANO2IMS) error {
+	// Create the service account:
+	sa, err := r.createServiceAccount(ctx, orano2ims, utils.ORANO2IMSDeploymentManagerServerName)
+	if err != nil {
+		return err
+	}
+
+	// The role and binding are cluster level objects, so to make them unique we use the
+	// namespace as a prefix:
+	roleName := fmt.Sprintf("%s-%s", orano2ims.Namespace, utils.ORANO2IMSAlarmSubscriptionServerName)
+	bindingName := fmt.Sprintf("%s-%s", orano2ims.Namespace, utils.ORANO2IMSAlarmSubscriptionServerName)
+
+	// Grant the service account permissions to read the objects it needs, for example managed
+	// clusters:
+	role := &rbacv1.ClusterRole{
```

Comment:
I think it would be a good idea to use the `utils.CreateK8sCR` function for creating these new resources for 2 reasons:
1. It sets the owner resource, so when the `orano2ims` CR is deleted, these resources will also be deleted.
2. If anyone is to update these resources, the operator will PATCH/UPDATE them to the expected values so that everything works as expected.

Please ignore these if they are not relevant to what you had in mind.

Reply:
I agree, will do it.